server/lib/notifications/agents/discord.ts

```diff
@@ -98,106 +98,62 @@ class DiscordAgent
 
     const fields: Field[] = [];
 
+    fields.push({
+      name: 'Requested By',
+      value: payload.notifyUser.displayName ?? '',
+      inline: true,
+    });
```

Comment:
Not all notifications are requests (test notification, currently, and more in the future)